internal/testutils/jimmtest/gorm.go

```diff
@@ -322,7 +322,7 @@ func getOrCreateTemplateDatabase() (string, string, error) {
 
 	templateName, templateDSN, err := createTemplateDatabase()
 	if err != nil {
-		return "", "", errors.E(err, "error creating template database")
+		return "", "", fmt.Errorf("error creating template database: %w", err)
```

Comment:
hmm.. yes.. i'm getting rather annoyed by our errors package.. not really fond of the way it handles errors..